.dockerignore

```diff
@@ -0,0 +1,13 @@
+################################################################################
+# Repo
+
+.git/*
+.dockerignore
+.gitignore
+Dockerfile
```

Comment:
```suggestion
Dockerfile
CONTRIBUTING.md
LICENSE.txt
MIGRATION.md
README.md
```
this improves caching, right?

Reply:
I the `.md` and `.txt` files are inconsequential to the build, perhaps is worth blanketing them all in the root directory?
```
*.md
*.txt
```

Reply:
sounds good to me